src/arrays.js

```diff
@@ -1,61 +1,93 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index < array.length) {
+    return array[index];
+  }
+  if (index >= array.length) {
+    return array[index - array.length];
+  }
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  const newArray = [];
+
+  for (let i = 0; i < strings.length; i++) {
+    newArray.push(strings[i].toUpperCase());
+  }
+  return newArray;
```

Comment:
While this does work, there is a cleaner way using a `map` rather than a for loop:
```suggestion
  return strings.map(string => {
    return string.toUpperCase();
  });
```
You're doing something very similar with reverseWordsInArray just below. Incidentally you don't need a higher order function here, there are built in methods in javascript as seen here that will handle this just fine